### DISCUSSION TITLE
Usernames

### DISCUSSION THREAD

**Discussion by psh4nk** [psh4nk/Freenove_4WD_Smart_Car_Kit_for_Raspberry_Pi#3] [General]
Make a comment below with your username and team name so I can add you as a collaborator on this repo.

**Comment by shan131** [comment]
shan131

**Comment by car-radio** [comment]
username: car-radio 
team: MC
Carmen and Steven

**Comment by ahoang928** [comment]
username: ahoang928
team name: Mymon
Angel and Ricardo

**Comment by RickyArmon** [comment]
Username: RickyArmon
Team name:  Mymon (Ricardo and Angel)

**Comment by spumel534** [comment]
username: spumel534 
team name: MC
Steven and Carmen

**Comment by Twait-Waffle** [comment]
Austen&Ryan
This is Austen

**Comment by 1medlinj** [comment]
Jacob Medlin & Daniel Grigsby
Username: 1medlinj
Team Name: Jacob_Daniel

**Comment by Rbulharowski** [comment]
username: Rbulharowski
team: Austen&Ryan

**Comment by carloscuau10** [comment]
Username: carloscuau10
Team Name: LaserEyes

**Reply by carloscuau10** [reply]
Carlos and Zack

**Comment by Andrew-Shank** [comment]
username: Andrew-Shank
team with: Sean Jackson
team name: js

**Comment by Jputerbaugh** [comment]
Username: jputerbaugh
Team Name: jcpenney

**Comment by danielgrigsby03** [comment]
Username: danielgrigsby03
Team Name: Jacob_Daniel
Teammate:  Jacob

**Comment by D3l1rium** [comment]
username m3ssm0r3m
Team Name : png
Actual name: Peyton Coleman
;)

**Comment by atlas-77** [comment]
name: Nathan Johnson
partner's name: Peyton Coleman
username: atlas-77
team name: png

**Comment by ccrawf** [comment]
username: ccrawf
team name: jcpenney

**Reply by ccrawf** [reply]
Chris Crawford and Justin Puterbaugh

**Comment by jackwl7** [comment]
japi (teamname)

**Comment by SilverXVI** [comment]
Username: SilverXVI
Team Name: Bandrew_Satten
My Name: Andrew Silver
Teammate: Barrett Batten

**Comment by jackwl7** [comment]
John Lundy (Jack), JaNyia Young 

teamname: japi

**Reply by Andrew-Shank** [reply]
good luck, you need it

**Comment by KingSean18** [comment]
username: KingSean18
team name: js 
name: Sean Jackson

**Reply by Andrew-Shank** [reply]
to go fast is to go well

**Comment by CLOAKERWAVE** [comment]
This is Zach
CLOAKERWAVE
LaserEyes

**Comment by jackwl7** [comment]
This is John (Jack) Lundy and JaNyia Young
jackwl7 on Github
teamname: japi
